scripts/db_migrator.py

```diff
@@ -125,6 +126,43 @@ def migrate_interface_table(self):
                 self.configDB.set_entry(table, key[0], data[key])
                 if_db.append(key[0])
 
+    def migrate_mgmt_ports_on_s6100(self):
+        '''
+        During warm-reboot, add back two 10G management ports which got removed from 6100
+        to ensure no change in bcm.config from older image
+        '''
+        if device_info.is_warm_restart_enabled('swss') == False:
+            log.log_notice("Skip migration on {}, warm-reboot flag not set".format(self.hwsku))
+            return True
+
+        entries = {}
+        entries['Ethernet64'] = {'alias': 'tenGige1/1', 'description': 'tenGige1/1', 'index': '64', 'lanes': '129', 'mtu': '9100', 'pfc_asym': 'off', 'speed': '10000'}
+        entries['Ethernet65'] = {'alias': 'tenGige1/2', 'description': 'tenGige1/2', 'index': '65', 'lanes': '131', 'mtu': '9100', 'pfc_asym': 'off', 'speed': '10000'}
```

Comment:
Could you change the alias name to tenGigE instead of tenGige?.

Reply:
Done